pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "aequitas"
+version = "0.1.0"
+description = ""
+authors = ["None"]
+readme = "README.md"
+packages = [
+    { include = "aequitas", from="src" },
+    { include = "aequitas_cli", from="src" },
+    { include = "aequitas_webapp", from="src" },
+]
+
+[tool.poetry.dependencies]
+python = "3.9.16"
```

Comment:
I've tried installing on my side, and this was giving trouble, I changed it to: `python = ">=3.8,<4"`, some package had a requirement to be less than python version 4 somehow :sweat_smile: